frontend/public/locales/en-US/help.json

```diff
@@ -1,9 +1,98 @@
 {
-  "dashboard": {
+  "dashboardsInfo": {
     "title": "Dashboard",
-    "description": "Dashboard description",
+    "description": "A dashboard is a read-only snapshot of an analysis that you can share with other users for reporting purposes. By default, the solution provides an User Life Cycle dashboard to help you understand each stage in the customer jorney. You can also create custom dashboard by saving explorative analysis that you want to view regularly.",
     "links": {
-      "dashboardDocLinkName": "What is dashboard?"
+      "docLinkName": "What is dashboard?",
+      "docLink": "https://awslabs.github.io/clickstream-analytics-on-aws/en/latest/dashboard/"
```

Comment:
I prefer to build the documentation link like v1, using docs.aws.amazon.com for the English version. So specifying the relative here, then the frontend to generate the final link.

Reply:
`v1.0.x` or `v1.1.x` ? below is the document link sample:
https://awslabs.github.io/clickstream-analytics-on-aws/zh/1.0.x/solution-overview/
https://awslabs.github.io/clickstream-analytics-on-aws/zh/1.1.x/solution-overview/

Reply:
Or use `"solution_version": "v1",` in aws-exports.json?

Reply:
Good proposal! Let's use the major and minor versions from `aws-exports.json`.